src/test/java/posters/pageobjects/pages/user/LoginPage.java

```diff
@@ -35,9 +35,10 @@
 
     @Override
     @Step("ensure this is a login page")
-    public void isExpectedPage()
+    public LoginPage isExpectedPage()
     {
         loginForm.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.